tests/src/test/scala/cats/tests/EqTests.scala

```diff
@@ -0,0 +1,24 @@
+package cats
+package tests
+
+import org.scalatest._
+
+import cats.functor._
+
+class EqTests extends FunSuite {
+  {
+    import cats.implicits._
+    implicitly[Invariant[Eq]]
```

Comment:
Just nitpicking, but we don't need `implicitly` in all these tests as simulacrum generates an `apply` in the companion object. Should we just write `Invariant[Eq]`, `Invariant[Monoid]`, ...?